packages/ilios-common/addon/classes/yup-validations.js

```diff
@@ -232,3 +235,37 @@ function moreThan() {
     };
   };
 }
+
+function lessThan() {
+  return (validationParams) => {
+    //our current translations expects this key to be named gt and not more as it is in yup
+    const lt = validationParams.less;
+    return {
+      path: validationParams.path,
+      messageKey: 'errors.lessThan',
+      values: { lt },
+    };
+  };
+}
+
+function minimum() {
```

Comment:
This one seems like It might not be needed, is it min in both places?

Reply:
[string.min()](https://yup-docs.vercel.app/docs/Api/types#stringminlimit-number--ref-message-string--function-schema)
 is different from [number.min()](https://yup-docs.vercel.app/docs/Api/types#numberminlimit-number--ref-message-string--function-schema) - min chars  vs lowest number in range. the error message is different, too.
 
agreed, the naming of these two functions is not ideal. as a follow-up item, i'd like to clean up the yup-validations class - perhaps inline all these one-off functions, or scope/namespace them better. not entirely sure yet.